Gruntfile.js

```diff
@@ -72,6 +73,15 @@ module.exports = function(grunt) {
             server: {
                 args: ["server.js"]
             }
+        },
+        watch: {
```

Comment:
This isn't being called in any of the registered tasks? I assume it should watch for changes in server.js and then rerun the server when it detects a change? What do I run to see that in action?